### PR TITLE
[MB-2795] Support API List MTOs Endpoint

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Adds support endpoint to list all move task orders

## Setup
`make server_run`
`prime-api-client support-list-mtos --insecure | jq`

## Code Review Verification Steps
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-2795) for this change

### CODE REVIEW

File: pkg/handlers/supportapi/internal/payloads/model_to_payload.go [hunk 1]
```diff
@@ -12,6 +12,16 @@ import (
 	"github.com/transcom/mymove/pkg/models"
 )
 
+// MoveTaskOrders payload
+func MoveTaskOrders(moveTaskOrders *models.MoveTaskOrders) []*supportmessages.MoveTaskOrder {
```
Comment (shimonacarvalho) [inline pkg/handlers/supportapi/internal/payloads/model_to_payload.go:16]: Does this nest the payment requests and service items and such?

The ticket requests "This should match the structure of the fetch-mto-updates endpoint in the Prime API - including populating all of the nested items for MTOs."

File: swagger/support.yaml [hunk 2]
```diff
@@ -18,6 +18,37 @@ schemes:
   - http
 paths:
   '/move-task-orders':
+    get:
+      summary: listMTOs
+      description: >
+        Gets all move task orders.
+        Provides all move task orders that are have both been made available to prime and not made available to prime.
```
Comment (shimonacarvalho) [inline swagger/support.yaml:25]: "that are have been" sounds awkward. Maybe "Returns all move task orders, regardless of whether they have been made available to prime"

Comment (Ronolibert) [reply swagger/support.yaml:25]: Woops! Will make this correction

File: pkg/handlers/supportapi/internal/payloads/model_to_payload.go (hunk 1 above)
Comment (Ronolibert) [reply pkg/handlers/supportapi/internal/payloads/model_to_payload.go:16]: I missed that! Will add service items as well

Comment (shimonacarvalho) [reply pkg/handlers/supportapi/internal/payloads/model_to_payload.go:16]: Also if you are using the same function for get-mto and list-mtos, it's good for get-mto to get all that nested stuff too, if it comes for free. If not, no worries make a separate ticket for us to add that to get-mto.

Comment (Ronolibert) [reply pkg/handlers/supportapi/internal/payloads/model_to_payload.go:16]: Good catch! Was pretty much for free, added it on

Comment (shimonacarvalho) [reply pkg/handlers/supportapi/internal/payloads/model_to_payload.go:16]: Awesome!

**Pull request merged**